File: Operating-System/Process/Create_Process/fork.c
#include <sys/types.h>   /*Header file containing fork defination*/
#include <unistd.h>       /*Header file containing fork defination*/
#include <stdio.h>
int main()
{
		pid_t fd;					/*data type for process id pid_t process identification and it is used to represent process id*/
		fd =fork();					/*calling fork system call to make new process*/
		if(fd ==0)					/*fd ==0 for child process && fd>0 for parent processs fd <0 process not created error*/
		{
	
			printf("I am a child process\n My process id is %ld\n",(long)getpid());				/*getpid() used to get id of current running process*/
			printf("My parent is %ld\n",(long)getppid());		                                        /*getppid() used to get pid of the parent process of the child process*/
		}
		else
			printf("I am a parent process\n Myprocess id is %ld", (long)getpid());
		
}

      

File: Operating-System/Process/Create_Process/fork.py
# Python code to create child process 
import os 

def parent_child(): 
	n = os.fork() 

	# n equals to zero  means that is child process 
	if n == 0: 
		print"Child process and id is : ", os.getpid()
                print"Child process parent is : ", os.getppid()
            

	# n  greater than 0 means a parent process
	else: 
		print"Parent process and id is : ", os.getpid()
		
# Driver code 
parent_child() 

